src/components/App.js

```diff
@@ -15,6 +15,7 @@ fontawesome.library.add(brands, faHome, faPlusSquare, faCoffee, faFutbol, faLeaf
 
 class App extends Component {
   render() {
+    document.body.style.background = '#656f7d';
```

Comment:
shouldn't this be in a variable?

Reply:
or... why not in sasS?

Reply:
@Br3nda yep this was done because I needed to dynamically update the body background which is inside of index.html
If you notice, the home page, listing page and service page all have different body background colours.